tests/test_torch_nn.py

```diff
@@ -184,6 +184,28 @@ def test_torch_nn_Dropout():
   assert jnp.mean(res_jax == res_torch.numpy(force=True)) > 0.5
 
 
+def test_torch_nn_PReLU():
+  model = torch.nn.PReLU(3)
+  input_batch = random.normal(random.PRNGKey(123), (1, 3, 112, 112))
+  params = {k: 0.1 * random.normal(random.PRNGKey(123), v.shape) for k, v in model.named_parameters()}
+  model.load_state_dict({k: j2t(v) for k, v in params.items()})
+  res_torch = model(j2t(input_batch))
+
+  jaxified_module = t2j(model)
+  res_jax = jaxified_module(input_batch, state_dict=params)
+  res_jax_jit = jit(jaxified_module)(input_batch, state_dict=params)
+
+  # Test forward pass without and with jit
+  aac(res_jax, res_torch.numpy(force=True), atol=1e-5)
+  aac(res_jax_jit, res_torch.numpy(force=True), atol=1e-5)
+
+  # Test gradients
+  jax_grad = grad(lambda p: (jaxified_module(input_batch, state_dict=p) ** 2).sum())(params)
+
+  res_torch.pow(2).sum().backward()
+  aac(jax_grad["weight"], model.weight.grad, atol=1e-3)
```

Comment:
let's also test the gradient on `input`. is there a clean way to test that here? if not you can test `torch.nn.functional.prelu` directly and `t2j_function_test` will handle it for you